ceph/ceph.py

```diff
@@ -607,9 +607,12 @@ def get_osd_metadata(self, osd_id, client=None):
                 return metadata
         return None
 
-    def osd_check(self, client):
+    def osd_check(self, rhbuild, client):
```

Comment:
Kindly add docstrings... I am aware that it wasn't available earlier but it would be helpful if we could add it.

Reply:
sure

Reply:
done